pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "laser_encoders"
 version = "0.0.1"
 authors = [{name = "Facebook AI Research"}]
 description = "LASER  Language-Agnostic SEntence Representations is a toolkit to calculate multilingual sentence embeddings and to use them for document classification, bitext filtering and mining"
-readme = "README.md"
+readme = "laser_encoders/README.md"
```

Comment:
Is it necessary to prefix with `laser_encoders`?

Reply:
Yea, Still because of testpypi. If you don't point to it, it would use the README of the main laser repo which doesn't contain any information on how to run the laser package

Reply:
Do we not want the `pyproject.toml` file to live inside `laser_encoders`?

Reply:
Well I can't say for sure, you guys get to decide that. I worked based on the present structure. But my own suggestion too would be to move the `pyproject.toml` file

Reply:
Yes, let's move it into `laser_encoders`

Reply:
Alright then